docs/events/glides/com.mbta.ctd.glides.trips_updated.v1.mdx

```diff
@@ -0,0 +1,288 @@
+
+_Notes:_
+
+- setting a new location or time does not modify the [TripKey](#tripkey) for a scheduled trip.
+- If `cars` is updated from a 1-element list to a 2-element list, then Glides SHOULD include data for the 2nd car. This is relevant if a two-car train has a car removed and then re-added. If a field was set before the car was dropped, then when the car is restored, clients MUST assume that the field is `"none"` as opposed to retaining its previous value, but Glides SHOULD include the data to avoid the ambiguity.
+- Some fields, such as `startTime` or `cars`, are not relevant to dropped trips. Those fields SHOULD NOT be set in an update that drops a trip or any following updates, until the trip is undropped by setting `dropped` to `false`. However, if a trip is undropped, clients MUST assume all fields retain their values from before the field was undropped. Clients MUST NOT ignore updates to fields when a trip is dropped, even if they aren't relevant for dropped trips. For example, if a trip is dropped, and then the `startTime` is updated, and then it's undropped, the trip's `startTime` is the value set while the trip was dropped. Glides MAY include extra fields in an update that undrops a trip, if they are changed as part of the same event that undropped the trip, or just to remind clients about them now that they're relevant.
+
+## Examples
+
+<Examples
+  examples={[
+    {
+      label: "15 Minute Delay",
+      description: (
+        <>
+          <p>
+            Inspector Alice (badge number: 123) is working at Mattapan. The
+            1:30am trip is being held 15 minutes to wait for a final Red Line
+            train.
+          </p>
+          <p>
+            Note that the date of the the event is 2023-01-23, but the service
+            date is 2023-01-22. The hour in the time itself is greater than 24
+            hours, to reflect that the time is the next calendar day.
+          </p>
+        </>
+      ),
+      json: exampleDelay,
+    },
+    {
+      label: "Dropped Trip and Manage Headways",
+      description: (
+        <>
+          <p>
+            There are four scheduled trips going to Government Center: 9:55am,
+            10:00am, 10:05am, and 10:10am.
+          </p>
+          <p>
+            Due to staffing issues, Inspector Alice drops the 10:05am trip,
+            along with its return trip. (First event with two updated trips).
+          </p>
+          <p>
+            Using the "Manage Headways" feature, Alice changes the expected
+            departure times of the remaining trips to 9:56am, 10:02am, and
+            10:08am. (Second event with three updated trips).
+          </p>
+          <aside>
+            (Scheduled data is unrealistically abbreviated for this example.
```

Comment:
suggestion: the first example also has abbreviated schedules, but does not have this note.

Reply:
Good catch. I've added more a more realistic schedule to that one.

(I added the data instead of the note because it's the first and most basic example so it's more important that it's realistic, and because it was only one car so it's not distracting from the rest of the data.)